app/views/layouts/devise/sessions.html.erb

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>

```

Comment:
Since this is pretty much the same as the `application` layout, is there any reason why we need this? Also, it's missing the notices, which will be needed for errors.